### PR TITLE
Updated APNS related documentation for 2017-04-01 version

### PR DESCRIPTION
**Pull request by stankovski**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-04-01/notificationhubs.json [hunk 1]
```diff
@@ -1898,38 +1898,38 @@
       "properties": {
         "apnsCertificate": {
           "type": "string",
-          "description": "The APNS certificate."
+          "description": "The APNS certificate. Specify if using Certificate Authentication Mode"
```
Comment (zhangeugenia) [inline specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-04-01/notificationhubs.json]: [nit] Missing period on descriptions ending with `Specify if using * Authentication Mode`

File: specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-04-01/notificationhubs.json [hunk 2]
```diff
@@ -1898,38 +1898,38 @@
...
         },
         "certificateKey": {
           "type": "string",
-          "description": "The certificate key."
+          "description": "The certificate password key if exists."
```
Comment (zhangeugenia) [inline specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-04-01/notificationhubs.json]: [nit] `...if exists` to `...if it exists`

File: specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-04-01/notificationhubs.json [hunk 3]
```diff
@@ -1898,38 +1898,38 @@
...
+          "description": "The issuer (iss) registered claim key. The value is a 10-character TeamId, obtained from your developer account. Specify if using Token Authentication Mode"
         },
         "token": {
           "type": "string",
-          "description": "Provider Authentication Token, obtained through your developer account"
+          "description": "Provider Authentication Token, obtained through your developer account. Specify if using Token Authentication Mode"
         }
       },
-      "description": "Description of a NotificationHub ApnsCredential."
+      "description": "Description of a NotificationHub ApnsCredential. Note that there is no explicit switch between Certificate and Token Authentication Modes. The mode is determined based on the properties being passed."
```
Comment (zhangeugenia) [inline specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-04-01/notificationhubs.json]: [nit] `...the properties being passed.` to `...the properties passed in.` or `...the properties being passed in.`

**Review by zhangeugenia — APPROVED** [review]
All nit comments, the changes looks good 😀 ✨

**Comment by stankovski** [comment]
Addressing feedback from https://github.com/MicrosoftDocs/azure-docs/issues/34922

**Review by markcowl — APPROVED** [review]

**Comment by stankovski** [comment]
Thanks @markcowl for reviewing. What do I need to do to get the PR merged?

**Comment by markcowl** [comment]
@stankovski just need the required checks to pass.  Reunning and will investigate failures

**Comment by stankovski** [comment]
@markcowl I just realized that this PR is still open. Any way to get it merged?

**Pull request merged**